client/src/main/scala/skuber/json/package.scala

```diff
@@ -105,6 +105,40 @@ package object format {
     }
   }
 
+  implicit val seccompProfileFmt: Format[Security.SeccompProfile] = new Format[Security.SeccompProfile] {
+
+    override def reads(json: JsValue): JsResult[Security.SeccompProfile] = json match {
+      case JsObject(fields) =>
+        fields.get("type") match {
+          case Some(JsString("Unconfined")) =>
+            JsSuccess(Security.UnconfinedProfile())
+          case Some(JsString("RuntimeDefault")) =>
+            JsSuccess(Security.RuntimeDefaultProfile())
+          case Some(JsString("Localhost")) =>
+            val profileConfigPath: String = fields("localhostProfile").as[String]
+            JsSuccess(Security.LocalhostProfile(profileConfigPath))
+          case operator => JsError(s"Unknown Seccomp profile '$operator'")
+        }
+
+      case _ => JsError(s"Unknown Seccomp")
```

Comment:
please treat the error in a more resilient way, you can return Option[SeccompProfile] and in that case None or you can configure a profile that is "Unknown"